EmpleoDotNet/Views/Shared/_JobListPartial.cshtml

```diff
@@ -9,7 +9,7 @@
             <a href="@Url.Action("Detail", "JobOpportunity" , new { id=UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)})" class="featured">
                 <div class="row">
                     <div class="col-md-1 hidden-sm hidden-xs text-center">
-                        @if (!string.IsNullOrEmpty(jobOpportunity.CompanyLogoUrl))
+                        @if (UrlHelperExtensions.IsImageAvailable(jobOpportunity.CompanyLogoUrl))
```

Comment:
I think this is performance-less, `IsImageAvailable()` method is making an http request inside a foreach.